************* Module pip._internal.resolution.resolvelib.factory
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:72:4: R0903: Too few public methods (0/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:79:0: C0103: Class name "C" doesn't conform to PascalCase naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:89:0: R0902: Too many instance attributes (14/7) (too-many-instance-attributes)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:90:4: R0913: Too many arguments (10/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:167:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:193:16: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:208:16: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:218:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:347:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:479:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:600:4: C0103: Argument name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:600:4: R0912: Too many branches (13/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py:24:0: W0611: Unused ResolutionImpossible imported from pip._vendor.resolvelib (unused-import)

-----------------------------------
Your code has been rated at 9.58/10

